client_subscribe.go

```diff
@@ -0,0 +1,102 @@
+package sdk
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+const reconnectDelay = 2 * time.Second
+
+var subscribeMessage = `{
+	"jsonrpc": "2.0",
+	"method": "subscribe",
+	"id": 0,
+	"params": {
+		"query": "tm.event='Tx' AND wasm._contract_address CONTAINS '%s' AND wasm.action='execute_claim'"
+	}
+}`
+
+var unsubscribeMessage = `{
+	"jsonrpc": "2.0",
+	"method": "unsubscribe",
+	"id": 0,
+}`
+
+func (c *Client) Subscribe(ctx context.Context, contractAddress string, acknowledge func(ctx context.Context, msg SubscribeMessage) error) error {
```

Comment:
does it makes sense to implement own ws connection and receive msgs? What do you think about my previos idea, where you use the existing ws logic

https://github.com/spell-club/sei-sdk/pull/4/files#diff-933802e60456833941ddf238159a46e07300456628c6981a6ce40428f35d709bR87

```
tmClient := сlient.GetTendermintClient()
tmClient.Subscribe()
```

Reply:
same story for events logic

Reply:
Yep, batch events in progress, will use this client

Reply:
Done!